docs/2_5_fmu_distribution.adoc

```diff
@@ -185,7 +185,7 @@ The following table lists the most common platform tuples for shared libraries a
 
 If runtime libraries are needed by the FMU that have to be present on the target machine and cannot be shipped within the FMU (e.g., due to licensing issues), then automatic processing is likely impossible.
 In such cases special handling is needed, for example, by providing the runtime libraries at appropriate places by the receiver.
-The requirements and the expected processing should be documented in the `documentation` directory in this case. +
+The requirements and the expected processing must be documented in the `documentation` directory in the file `externalDependencies.{txt|html}`. +
```

Comment:
The alternative is to say: "Let´s not define things, people will not do it right anyway."
Poorly documented FMUs that are not running because the user does not know what to do *should* be blamed on the exporter - and this "must" is a blame distribution mechanism.
In the end, we express our expectation, knowing about the reality...